test/SecTester.Repeater.Tests/SecTester.Repeater.Tests.csproj

```diff
@@ -10,7 +10,6 @@
   </ItemGroup>
 
   <ItemGroup>
-    <Folder Include="Api"/>
```

Comment:
Please revert back if folder is still there

Reply:
It does not exist anymore.